lib/matplotlib/sphinxext/plot_directive.py

```diff
@@ -151,6 +151,24 @@
         The plot_srcset option is incompatible with *singlehtml* builds, and an
         error will be raised.
 
+    plot_rst_directory
+        By default, the directory for RST files is automatically determined
+        from the name of the Python file. This works fine with, e.g., the
+        ``sphinx.ext.autodoc`` extension. However, you may need to modify the
+        path if RST files are generated in another directory. This is the
+        case if you use the ``autodoc2`` extension.
+
+        For it to work, you need to set the ``plot_rst_directory`` option to:
+
+        .. code:: python
+
+            plot_rst_directory = "source/apidocs/<your_package>/"
+
+        .. node::
```

Comment:
```suggestion
        .. note::
```

Was this meant to be note?

Reply:
Indeed, good catch!